docs/Tasks/explore-available-packages.md

```diff
@@ -1,135 +1,141 @@
-  resourceVersion: "3765"
-  uid: 43396920-4af4-4daf-a069-be68b8a0631e
-spec:
-  catalog:
-    name: operatorhubio
-  channels:
-  - entries:
-    - name: argocd-operator.v0.0.11
-      replaces: argocd-operator.v0.0.9
-    - name: argocd-operator.v0.0.12
-      replaces: argocd-operator.v0.0.11
-    - name: argocd-operator.v0.0.13
-      replaces: argocd-operator.v0.0.12
-    - name: argocd-operator.v0.0.14
-      replaces: argocd-operator.v0.0.13
-    - name: argocd-operator.v0.0.15
-      replaces: argocd-operator.v0.0.14
-    - name: argocd-operator.v0.0.2
-    - name: argocd-operator.v0.0.3
-      replaces: argocd-operator.v0.0.2
-    - name: argocd-operator.v0.0.4
-      replaces: argocd-operator.v0.0.3
-    - name: argocd-operator.v0.0.5
-      replaces: argocd-operator.v0.0.4
-    - name: argocd-operator.v0.0.6
-      replaces: argocd-operator.v0.0.5
-    - name: argocd-operator.v0.0.8
-      replaces: argocd-operator.v0.0.6
-    - name: argocd-operator.v0.0.9
-      replaces: argocd-operator.v0.0.8
-    - name: argocd-operator.v0.1.0
-      replaces: argocd-operator.v0.0.15
-    - name: argocd-operator.v0.2.0
-      replaces: argocd-operator.v0.1.0
-    - name: argocd-operator.v0.2.1
-      replaces: argocd-operator.v0.2.0
-    - name: argocd-operator.v0.3.0
-      replaces: argocd-operator.v0.2.1
-    - name: argocd-operator.v0.4.0
-      replaces: argocd-operator.v0.3.0
-    - name: argocd-operator.v0.5.0
-      replaces: argocd-operator.v0.4.0
-    - name: argocd-operator.v0.6.0
-      replaces: argocd-operator.v0.5.0
-    name: ""
-  defaultChannel: ""
-  description: ""
-  icon:
-    data: PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2Z==
-    mediatype: image/svg+xml
-  packageName: argocd-operator
-status: {}
-```
-
-**This CR is most helpful when exploring the versions of a package that are available for installation on cluster, and the upgrade graph of versions** (eg if v0.5.0 of `argocd-operator` is installed on cluster, what is the next upgrade available? The answer is v0.6.0).
\ No newline at end of file
+After you add a catalog of exetensions to your cluster, you must port forward your catalog as a service.
+Then you can query the catalog by using `curl` commands and the `jq` CLI tool to find extensions to install.
+
+## Prerequisites
+
+* You have added a catalog of extensions, such as [OperatorHub.io](https://operatorhub.io), to your cluster.
+* You have installed the `jq` CLI tool.
+
+## Procedure
+
+1. Port forward the catalog server sevice:
+
+    ``` terminal
+    $ kubectl -n catalogd-system port-forward svc/catalogd-catalogserver 8080:80
+    ```
+
+2. Return a list of all the extensions in a catalog:
+    ``` terminal
+    $ curl http://localhost:8080/catalogs/operatorhubio/all.json | jq -s '.[] | select(.schema == "olm.package") | .name'
+    ```
+
+    ??? success
```

Comment:
what does `???` provide?

Reply:
Sorry, I had a draft for this comment, but I forgot to submit it. 

The `???` creates a code block with a toggle.